seaworthy/pytest/fixtures.py

```diff
@@ -54,8 +54,9 @@ def resource_fixture(definition, name, scope='function'):
     """
     Create a fixture for a resource.
 
-    Note that it is important to keep a reference to the fixture function
-    returned by this function::
+    Note that this function returns a fixture function. It is important to keep
+    a reference to the returned function within the scope of the tests that use
+    the fixture::
```

Comment:
Should this (and others like it) be a `.. note::` block?

Reply:
I did not know about this. Giving it a go.